master/internal/command/events.go

```diff
@@ -135,13 +134,23 @@ func (e *eventManager) Receive(ctx *actor.Context) error {
 func canAccessCommandEvents(ctx *actor.Context, c echo.Context) error {
 	curUser := c.(*context.DetContext).MustGetUser()
 	taskID := model.TaskID(ctx.Self().Parent().Address().Local())
-	ownerID, err := db.GetCommandOwnerID(c.Request().Context(), taskID)
-	if err != nil {
+
+	// CHECK why did we go to the DB and not actor system? logs for terminated ntsc?
```

Comment:
I'm not sure what's being asked here... can you clarify?

Reply:
we used to always go to the DB just to get the command's owner id whereas it should be available in memory for non-terminal commands. 
I'd guess since we have to consult the DB in some cases we do this for simplicity?